lua/luasnip/session/snippet_collection.lua

```diff
@@ -20,17 +39,11 @@ local by_prio = {
 		order = {
 			1000,
```

Comment:
Just noticed: these can also be removed, now that the tables with prio 1000 are no longer present from the beginning

Reply:
Ah yes, to be honest, I didn't digged into how this order/store sorted worked (isn't that relevant for this PR, the only point of contact was removing the initial population of the tables though).
Should be fine by now (please check again).
(I'd like to keep the `order` member in the initial tables to indicate to any reader that this member exists (don't overuse the autocreating feature where there's no real benefit (`oder` will be created anyways soon) as it just makes reading/understanding much harder))

Reply:
`order` is just for quickly iterating the priorities in order.
Looks good, I definitely agree with keeping the empty tables 👍👍